src/utils.coffee

```diff
@@ -279,3 +280,15 @@ exports.validateKeys = (options, validSet) ->
 		return if !options?
 		invalidKeys = _.keys(_.omit(options, validSet))
 		throw new Error("Using #{invalidKeys.join(', ')} is not allowed.") if !_.isEmpty(invalidKeys)
+
+checkAndAddIptablesRule = (rule) ->
+	execAsync("iptables -C #{rule}")
+	.catch ->
```

Comment:
Ok, it's been a while since I've used it myself and just had memories of exit code checking so wanted to check :)